distribution/client/test/helpers.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * This module contains functions which are helpful for running across multiple
+ * tests
+ */
+const fs            = require('fs');
+
+class Helpers {
+  constructor () {
+  }
+
+  checkFileExists(filename) {
+    return new Promise(r=>fs.access(filename, fs.F_OK, e => r(!e)));
+  }
+
+  touchFile(filename) {
```

Comment:
sync `fs.openSync`, or even `fs.writeFileSync('/path/to/theFile', 'blah content')` might be easier for the purpose?